modules/apps/headless-apio/media-object/media-object-apio-client-test/src/testIntegration/java/com/liferay/media/object/apio/client/test/MediaOjectApioTest.java

```diff
@@ -0,0 +1,641 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.media.object.apio.client.test;
+
+import com.liferay.media.object.apio.client.test.internal.activator.MediaObjectTestActivator;
+import com.liferay.oauth2.provider.test.util.OAuth2ProviderTestUtil;
+import com.liferay.portal.apio.test.util.ApioClientBuilder;
+
+import java.io.File;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.hamcrest.Matchers;
+import org.hamcrest.core.IsNull;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.Archive;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Ruben Pulido
+ */
+@RunAsClient
+@RunWith(Arquillian.class)
+public class MediaOjectApioTest {
+
```

Comment:
In general, simplify test following the same patters we already have in StructuredContentApioTest and BlogsApioTest.

Extract test initialization to methods ...

Reply:
👍